source/PaginationTrait.php

```diff
@@ -25,6 +27,9 @@
      */
     public function pagination($total, $current, $perPage = null, $neighbours = null): Pagination
     {
-        return $this['pagination']($total, $current, $perPage, $neighbours);
+        /** @var Pagination $pagination */
```

Comment:
Every call that returns unknown type is highly recommended to be specified.
This helps static analysis and finding bugs. Otherwise PHP my auto cast it to something that author didn't expected. Not to mention code understanding, code completion and debugging.

Particularly this fixed helped fixing Travis CI issue. It's needed because that way PHPStan can compare the type after assignment and the function returned type.